data/migrations/20190214213828_categories.js

```diff
@@ -0,0 +1,10 @@
+exports.up = function(knex, Promise) {
```

Comment:
You incorporated you knex migrations and seeding scripts for your data model is normalized objective. The data I saw above that you have running for the dummy data looks good and appears to be working for all intent and purposes.